MantidPlot/make_package.rb.in

```diff
@@ -244,6 +257,40 @@ directories.each do |directory|
   addPythonLibrary("#{path}/#{directory}")
 end
 
+
+# ---------------------------------------------
+# H5Py section
+# ---------------------------------------------
+h5py_path = "/usr/local/lib/python2.7/site-packages"
+h5py_directories = ["h5py"]
+h5py_directories.each do |directory|
+  addPythonLibrary("#{h5py_path}/#{directory}")
+end
+
+h5py_patterns = ["**/h5py/*.so"]
+change_id(h5py_patterns, "Contents/MacOS/h5py/")
+
+h5py_linking_dir = find_linking_directories(h5py_patterns, 'lib/libhdf5', '/usr/local/opt/hdf5/lib')
+
+h5py_patterns.each do |pattern|
+  Dir[pattern].each do |library|
+    basename =  File.basename(library)
+    dependencies = `otool -L #{library}`
+    dependencies.split("\n").each do |dependency|
+        currentname = dependency.strip.split(" ")
+        filename = currentname[0]
+        if filename.include? "libhdf5"
+          name_split_on_slash = filename.strip.split("/")
+          filename_no_dir = name_split_on_slash[-1]
+          parts = filename_no_dir.strip.split(".")
+          loader_filename = parts[0] + "." + parts[2]
+          `install_name_tool -change #{h5py_linking_dir[0]}/#{filename_no_dir} @loader_path/../#{loader_filename} Contents/MacOS/h5py/#{basename}`
```

Comment:
`@loader_path` works, though I have been trying to standardize on the (slightly) more flexible `@rpath`. What do you think about instead using that syntax?